feconf.py

```diff
@@ -635,7 +632,6 @@ def get_empty_ratings():
     u'15': 'classifier_demo_exploration.yaml',
     u'16': 'all_interactions',
     u'17': 'audio_test',
```

Comment:
maybe add a comment to say why '18' is deprecated and should (presumably) not be reused.

Reply:
Done